### PR TITLE
Move BrickFragment up into the app

### LINKED ISSUE #107: Provide alternative to requiring extending BrickFragment
Looks like a lot of the stuff can be pulled into a BrickFragmentManager class w/ onCreateView and onDestroyView methods...

### PR DESCRIPTION
**Pull request by thusson13**
Fixes https://github.com/wayfair/brickkit-android/issues/107

### CODE REVIEW

File: BrickKit/bricks/src/androidTest/res/layout/text_brick_vm.xml [hunk 1]
```diff
@@ -20,7 +20,7 @@
             android:textColor="@android:color/black"
             android:textSize="31sp"
             android:background="@android:color/white"
-            android:padding="@dimen/default_brick_content_padding"
```
Comment (kenyee) [inline BrickKit/bricks/src/androidTest/res/layout/text_brick_vm.xml:23]: should this be eight_dp?

File: BrickKit/bricks/src/androidTest/res/layout/text_brick_vm_placeholder.xml [hunk 2]
```diff
@@ -14,8 +13,8 @@
             android:textColor="@android:color/black"
             android:textSize="31sp"
             android:background="@android:color/darker_gray"
-            android:padding="@dimen/default_brick_content_padding" />
```
Comment (kenyee) [inline BrickKit/bricks/src/androidTest/res/layout/text_brick_vm_placeholder.xml:17]: ditto this one?

**Review by kenyee — APPROVED** [review]

File: BrickKit/bricks/src/androidTest/res/layout/text_brick_vm.xml (hunk 1 above)
Comment (thusson13) [reply BrickKit/bricks/src/androidTest/res/layout/text_brick_vm.xml:23]: didn't feel worth it to add a resource for a test only layout

File: BrickKit/bricks/src/androidTest/res/layout/text_brick_vm_placeholder.xml (hunk 2 above)
Comment (thusson13) [reply BrickKit/bricks/src/androidTest/res/layout/text_brick_vm_placeholder.xml:17]: ditto this one

**Pull request merged**